[WD-12238] new webpage at /data/opensearch/managed

akbarkz opened the pull request:
## Done

Added new webpage according to this [copy doc](https://docs.google.com/document/d/19gcIoKEibXTvZCsmtYCSQBMnBUmNUJI6RaLoMHL2_TA/edit) and this [design](https://www.figma.com/design/uJUwbUAGDog4KCMivrMpnl/24.10-Data-Fabric?node-id=166-4958&t=e5exMaJ2QIAaTFEt-0).

## QA

- Open webpage at https://canonical-com-1317.demos.haus/data/opensearch/managed and check if it complies with the copy doc and the design.

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-12238

webteam-app commented:
### [<img src='https://assets.ubuntu.com/v1/6baef514-ubuntu-circle-of-friends-large.svg' height=32 width=32> Demo</img>](https://canonical-com-1317.demos.haus)
### [<img src='https://assets.ubuntu.com/v1/e512b0e2-jenkins.svg' height=32 width=32> Jenkins </img>](https://jenkins.canonical.com/webteam/job/start-demo/7350)
### [<img src='https://assets.ubuntu.com/v1/7144ec6d-logo-jaas-icon.svg' height=32 width=32> demos.haus </img>](https://demos.haus)

lyubomir-popov commented:
all images are lacking [the vanilla highlighted background](https://vanillaframework.io/docs/examples/patterns/image/container/highlighted?theme=light)

akbarkz commented:
@lyubomir-popov I addressed all your remarks, except one that is about having equal height for pricing cards. Could you please help me with that, cause using existing Vanilla utilities does not seem to be able to help.

lyubomir-popov commented:
can you restart the demo pls - I get a 404

lyubomir-popov commented:
- lines need to be as wide as the grey boxes:
![image](https://github.com/user-attachments/assets/63499770-7d0b-4cd2-bd77-5af79b581f91)
- text unnecessarily compressed hjere, causing excessive wrapping. Can we give it one more column?
![image](https://github.com/user-attachments/assets/e77ec45f-97c7-4c8d-9ce8-62829f02e241)

For the equal heights, since we don't have a component for this in vanilla (we only seem to use this on the u.c site - is that correct @Sophie-32 ?) maybe just hardcoded min-heights? This is what I get:

![image](https://github.com/user-attachments/assets/3cc8abed-6f72-4d3c-8d34-a6e24294db29)

for the signpost logo, can we please use this file at 33px height, width set to auto:
https://assets.ubuntu.com/v1/ff455ff8-opensearch-logo--signpost.png

akbarkz commented:
@lyubomir-popov thanks for the review!
I replaced the logo and fixed the layout for it, I noticed it was behaving weirdly for smaller screens. 
As for cards, I added min-height as you suggested. Overall it looks good now, but on some smaller screen sizes they still differ in height. So please have a look and see if we're fine with that.

lyubomir-popov commented:
@akbarkz Just one question, this has a play icon, shouldn't it be clickable? 
![image](https://github.com/user-attachments/assets/d70c7f91-a3da-47a0-8c12-c82e3c9167a0)
The rest looks great!

anthonydillon commented on templates/data/opensearch/managed.html:
```diff
@@ -0,0 +1,447 @@
+        <hr class="is-muted" />
+        <a class="p-button--positive"
+           href="/contact-us"
+           aria-controls="data-opensearch-modal">Schedule a call</a>
```
This button doesn't work

akbarkz replied:
you're right, sorry, totally missed that. Will push a fix. Thanks for the review!

anthonydillon reviewed (approved):
Nice work @akbarkz, code looks good but one small comment about the CTA at the top of the page.

anthonydillon commented:
Should there be a link to this new page somewhere?
